pkgs/development/tools/kustomize/default.nix

```diff
@@ -2,8 +2,8 @@
 
 buildGoModule rec {
   pname = "kustomize";
-  version = "3.8.1";
-  # rev is the 3.8.1 commit, mainly for kustomize version command output
+  version = "3.8.2";
+  # rev is the 3.8.2 commit, mainly for kustomize version command output
   rev = "6a50372dd5686df22750b0c729adaf369fbf193c";
```

Comment:
We should remove this as this is gonna be wrong updated by @r-ryantm :sweat_smile: